dkms.in

```diff
@@ -3384,6 +3384,13 @@ if [[ ! $NO_WEAK_MODULES ]]; then
     esac
 fi
 
+# Execute post-transaction command if set and log its output
+execute_post_transaction() {
+    # Blank the log file before starting
+    : > "$dkms_tree/post_transaction.log"
+    invoke_command "$post_transaction ${kernelver[0]}" "Executing post-transaction command for kernel ${kernelver[0]}" "$dkms_tree/post_transaction.log" background
```

Comment:
`${﻿kernelver[0]}` should not be appended to the command line unconditionally. ﻿It should support allowing the user to decide whether to use $kernelver, just like the sign_file/mok_signing_key/mok_certificate settings.

Reply:
I think the main use case is for rebuilding the `initrd`, but anyway that's a good point, making the amendment.

Reply:
Added snippet like `sign_file/mok_signing_key/mok_certificate` to read the directive from the `framework.conf` file  so the `$kernelver` variable is read at the appropriate time.